### DISCUSSION TITLE
Matteo Collina

### DISCUSSION THREAD

**Discussion by just-be-dev** [devtools-fm/devtools.fm#228] [Announcements]
## Background

- https://github.com/mcollina
- https://nodeland.dev/
- https://twitter.com/matteocollina

Creator of
- [Fastify](https://www.fastify.io/), a web framework for Node.js
- [Pino](https://www.npmjs.com/package/pino), a super fast JSON logger
- [Mercurius](https://www.npmjs.com/package/mercurius), a GraphQL plugin for Fastify
- [Avvio](https://www.npmjs.com/package/avvio), the plugin system behind Fastify
- [fast-json-stringify](https://www.npmjs.com/package/fast-json-stringify), a fast JSON serializer

Founder and CTO of [platformatic](https://github.com/platformatic) (See [platformatic v1.0 launch](https://platformatic.dev/v1launch/))

Node TSC member, on the board of openJS Foundation

### OSS

- [x] How do you split your time between being a CTO, a lead maintainer on projects like fastify, and active member of the node TSC, and your other responsibilities?
- [x] You're a prolific open source creator. How'd you get your start in open source and what keeps you going?
- [x] What node features have you helped with?
- [x] The transition from cjs to esm node has been rough. What's your take on the current situation and how we might get out of it?
- [x] What is fastify? How does it differ from other node servers? 
- [x] What are some cool thing you can do with plugins?
- [x] What steps have you taken to make it fast?
- [ ] ~The plugin system is also its own package and you wrote it. What area some other things that have been built on it?~
- [ ] ~What is pino and how does it reduce the overhead of logging? Transports? Async?~

### Platformatic

- [x] What is platformatic?
- [x] Fastify is flexible and platformatic is opionated. What are those opinions?
- [x] What are some of the tools you've already shipped?

### Future

- [ ] ~What's your spiciest dev take?~
- [ ] What do you think the future of javascript runtimes is?

**Comment by hipstersmoothie** [comment]
https://github.com/tr1ckydev/hyperimport
https://github.com/privatenumber/pkgroll

**Comment by just-be-dev** [comment]
https://www.unison-lang.org/whats-new/cloud-teaser/
https://tanstack.com/form/latest/docs/guides/validation#adapter-based-validation
